plugins/module_utils/foreman_helper.py

```diff
@@ -454,6 +466,33 @@ def __init__(self, argument_spec=None, **kwargs):
             args.update(argument_spec)
         super(KatelloEntityApypieAnsibleModule, self).__init__(argument_spec=args, **kwargs)
 
+    def connect(self, ping=True):
+        super(KatelloEntityApypieAnsibleModule, self).connect(ping)
+        self._patch_subscription_upload_api()
+        self._patch_organization_update_api()
+
+    def _patch_subscription_upload_api(self):
```

Comment:
I am wondering whether we should put those in a separate file, as they start to clunk up the foreman_helper.